### DISCUSSION TITLE
Steam Deck support

### DISCUSSION THREAD

**Discussion by PGQT** [pop-os/pop#2725] [General]
With the new steam big picture mode basically bringing steam deck ui features to any desktop environment, it woud be incredible if we could get the steam deck's driver stack added into Pop! 
Other ideas are:
-Add better refinements for touch keyboard layout, which is very small, and hard to use on the deck.
-Work on better touch interface in general

I really like Pop!_os better than any other os, and i like cosmic better than any other DE, so getting these things worked out, would likely result in me running Pop! on every machine I've got.

**Comment by pirate-boop** [comment]
plzzz
